modules/ding_wayf_dk/modules/ding_wayf_dk_login/ding_wayf_dk_login.install.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * @file
+ * Handle installation processes.
+ */
+
+/**
+ * Implements hook_requirements().
+ */
+function ding_wayf_dk_login_requirements($phase) {
+  $requirements = array();
+
+  // Ensure translations don't break at install time.
+  $t = get_t();
+
+  if ($phase == 'runtime') {
+    if (!variable_get('wayf_hash', FALSE)) {
+      $requirements['wayf_hash'] = array(
+        'value' => $t('WAYF not configured correctly'),
```

Comment:
I would expect the value of the requirement to be `WAYF`, `Ding! WAYF` or something like that.